pipeline/train/requirements/train.in

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/hplt-project/OpusTrainer.git@9133e1525c7ee37f53ea14ee6a180152bf7ea192
```

Comment:
Why use a different version of OpusTrainer here?

Reply:
should be the same, but they fixed a bug that affected only running marian (argument parsing), I can sync them

Reply:
synced